### PR TITLE
#13: add support for valueLink (closes #13)

### LINKED ISSUE #13: add support for valueLink

### PR DESCRIPTION
**Pull request by FrancescoCioria**
Issue #13

### CODE REVIEW

**Comment by FrancescoCioria** [comment]
@gabro note that this PR breaks the previous API

**Comment by gabro** [comment]
what's the status of this?

**Comment by FrancescoCioria** [comment]
should be complete after a rebase. Today I'll check it

**Comment by FrancescoCioria** [comment]
@gabro please review and merge

File: test/tests/DatePickerInput-test.js [hunk 1]
```diff
@@ -106,4 +106,34 @@ describe('DatePickerInput', function() {
   });
 
 
+  it('DatePicker should work with valueLink', function() {
```
Comment (gabro) [inline test/tests/DatePickerInput-test.js]: as discussed offline, this test is not testing what it claims.
Please review it

**Comment by gabro** [comment]
Also, please provide a **test plan**

**Comment by FrancescoCioria** [comment]
I tested it using the last section of examples.js (after the subtitle "VALUE LINK") where I'm rendering a ```DatePickerInput``` using the ```valueLink``` key instead of ```value```.
The value is printed in the row above the ```DatePickerInput```

**Pull request merged**